lib/ZnapZend/ZFS.pm

```diff
@@ -410,6 +410,8 @@ sub sendRecvSnapshots {
         );
         #start forkcall event loop
         $fc->ioloop->start if !$fc->ioloop->is_running;
+        #wake up event loop
+        $fc->ioloop->one_tick();
```

Comment:
hmmm this whole section looks a bit odd to me ... have you looked at the state of `$fc->ioloop->is_running` ?  Because if this is NOT true, then my guess is that the reason for the problem you are seeing is that $fc is going out of scope at the end of the block and thus the event handlers are gone too ... with the one_tick call you allow the spam to happen ... but then it would make sense to make the is_running call into a propper if/then condition.